src/main/java/lotto/constant/message/ExceptionMessage.java

```diff
@@ -0,0 +1,26 @@
+package lotto.constant.message;
+
+import lotto.constant.LottoNumber;
+
+public enum ExceptionMessage {
+    ERROR("[ERROR] "),
```

Comment:
`enum`을 통해 에러 메시지를 관리하려고 한 시도는 너무 보기 좋습니다!

다만, 저는 `enum`을 "서로 연관된 상태를 나열하기 위해" 사용한다고 생각하는데요.
이 친구는 다른 "특정 예외 상황에 대한 메시지"라기 보단, 그냥 모든 예외 메시지의 앞에 붙는 전치사 같은 느낌이에요.

저였다면 이친구는 해당 클래스 내부에서 상수로 추출해서 사용했을 것 같습니다!
```suggestion
    private static final String PREFIX = "[ERROR] ";
    
    //
    public String getMessage(){
        return PREFIX + this.message;
    }
```

Reply:
enum 내부에서 상수로 추출해서 사용하는 방법도 있군요! 덕분에 배워요ㅎㅎ